string ROS_System
string action
string object
---
string answer